Bugzilla/Install/Requirements.pm

```diff
@@ -121,8 +121,9 @@ sub REQUIRED_MODULES {
     # versions prior to 3.008 are broken, see https://bugzilla.mozilla.org/show_bug.cgi?id=1560873
     {package => 'Template-Toolkit', module => 'Template', version => '3.008'},
 
-    # 1.300011 has a debug mode for SMTP and automatically pass -i to sendmail.
-    {package => 'Email-Sender', module => 'Email::Sender', version => '1.300011',},
+    # versions prior to 2.600 pulled Email::Address, we now use Email::Address::XS
+    {package => 'Email-Sender', module => 'Email::Sender', version => '2.600',},
+
```

Comment:
You should also add Email::Address::XS to the list, because maybe one day Email::Sender will no longer depend on Email::Address::XS and we would have a missing dependency check. Moreover, Email::Sender doesn't require any minimum version of Email::Address::XS, but you use parse_bare_address() and is_valid() in your code which are only available since version 1.01. IMO, the fix implemented in 1.05 can be seen as a security fix, so we should require 1.05 as a minimum version for Email::Address::XS (released Aug 7, 2022).

Reply:
That makes sense, apologies I misunderstood the earlier request.  I'll add back in the ::XS requirement, and your right, it makes sense to make it 1.05 minimum

Reply:
Done